packages/web-components/src/global/color-mode.css

```diff
@@ -1113,6 +1312,185 @@ ic-theme,
+    );
+    --ic-button-secondary-background-pressed-active: var(
+      --ic-action-default-bg-active
+    );
+    --ic-button-secondary-background-hover-monochrome: var(
+      --ic-action-dark-bg-hover
+    );
+    --ic-button-secondary-background-pressed-monochrome: var(
+      --ic-action-dark-bg-pressed
+    );
+    --ic-button-secondary-text: var(--ic-color-text-action-dark);
+    --ic-button-secondary-text-monochrome: var(--ic-color-text-primary-light);
+    --ic-button-secondary-text-active: var(--ic-color-text-action);
+    --ic-button-secondary-text-hover-active: var(
+      --ic-color-text-action-hover-dark
+    );
+    --ic-button-secondary-text-pressed-active: var(
+      --ic-color-text-action-pressed-dark
+    );
+    --ic-button-secondary-text-disabled: var(--ic-color-text-disabled-default);
+    --ic-button-secondary-text-disabled-monochrome: var(
+      --ic-color-text-disabled-default
+    );
+    --ic-button-secondary-text-hover-monochrome: var(--ic-action-dark-hover);
+    --ic-button-secondary-text-pressed-monochrome: var(
+      --ic-action-dark-pressed
+    );
+    --ic-button-secondary-icon-monochrome: var(--ic-color-icon-inverted);
+    --ic-button-secondary-icon-disabled: var(--ic-color-icon-disabled-dark);
+    --ic-button-secondary-icon-disabled-monochrome: var(
+      --ic-color-icon-disabled-dark
+    );
+    --ic-button-secondary-icon-hover-monochrome: var(
+      --ic-color-icon-hover-dark
+    );
+    --ic-button-secondary-icon-pressed-monochrome: var(
+      --ic-color-icon-pressed-dark
+    );
+    --ic-button-secondary-icon-active: var(--ic-color-icon-action-default-dark);
+    --ic-button-secondary-icon-hover-active: var(--ic-color-icon-action-hover);
+    --ic-button-secondary-icon-pressed-active: var(
+      --ic-color-icon-action-pressed
+    );
+    --ic-button-secondary-border: var(--ic-color-border-action-dark);
+    --ic-button-secondary-border-monochrome: var(
+      --ic-color-border-neutral-black
+    );
+    --ic-button-secondary-border-inverse: var(--ic-color-border-neutral-black);
+    --ic-button-secondary-border-disabled: var(
+      --ic-color-border-neutral-default
+    );
+    --ic-button-secondary-border-disabled-monochrome: var(
+      --ic-color-border-neutral-disabled-dark
+    );
+    --ic-button-secondary-border-hover: var(
+      --ic-color-border-action-hover-dark
+    );
+    --ic-button-secondary-border-pressed: var(
+      --ic-color-border-action-pressed-dark
+    );
+    --ic-button-secondary-border-hover-monochrome: var(
+      --ic-color-border-neutral-hover-dark
+    );
+    --ic-button-secondary-border-pressed-monochrome: var(
+      --ic-color-border-neutral-pressed-dark
+    );
+    --ic-button-tertiary-text-active: var(--ic-action-default-dark);
+    --ic-button-tertiary-icon-active: var(--ic-action-default-dark);
+    --ic-button-tertiary-text-disabled: var(--ic-color-text-disabled-default);
+    --ic-button-tertiary-icon-disabled: var(--ic-color-icon-disabled-dark);
```

Comment:
These icon tokens for when the button is disabled don't seem to be used anywhere. This doesn't really make a difference for primary or secondary variants as them and the text disabled tokens are the same color under the hood, however for tertiary the colors are differing from the designs.

I noticed this in the ic-horizontal-scroll, as the disabled buttons aren't appearing correctly, same with the active ones. Not sure what the actual issue is there in that case though.
![dm](https://github.com/user-attachments/assets/8a4db25d-79b2-4a37-9b9a-8ee121adac9a)
![dm_m](https://github.com/user-attachments/assets/832adc9b-4d2c-4b18-bd70-de720540f94a)